resources_portal/config/common.py

```diff
@@ -118,9 +118,9 @@ class Common(Configuration):
                 "format": "[%(server_time)s] %(message)s",
             },
             "verbose": {
-                "format": "%(levelname)s %(asctime)s %(module)s %(process)d %(thread)d %(message)s"
+                "format": "%(asctime)s %(levelname)s %(module)s %(process)d %(thread)d %(message)s"
```

Comment:
Absolutely, but do you want that for all logs or just the ones that go to cloud watch? Personally I find `{"message": "Something happened!", "time": "2015-09-01T06:06:26.524448", "level": "INFO"}` a lot less readable than `2015-09-01T06:06:26.524448 INFO Something happened!` when looking a logfile.

Either way, I think we can handle that with an additional issue because it looks like it might involve using the `django-log-formatter-json` library and getting that all configured can be another PR.

Reply:
I was thinking of the ones in CloudFront, I remember once that I tried to add kibana to our refinebio's logs but it was too complicated to parse them. Is there any drawbacks to using json for these?

Reply:
Ah ok. I think that makes sense and added a separate issue for it: https://github.com/AlexsLemonade/resources-portal/issues/39